packages/cms/pages/resources/resource/resource.tsx

```diff
@@ -463,9 +460,22 @@ export const Resource: React.FunctionComponent = () => {
                   <FilterList />
                 </SearchAndFilterContainer>
               </HeaderContainer>
-
-              <EuiSpacer size="xl" />
-
+              <EuiSpacer size="m" />
+              <EuiFlexGroup gutterSize="s" alignItems="center">
+                {filters.map((filter, idx) => (
+                  <EuiFlexItem grow={false} key={idx}>
+                    <EuiButton
+                      color="primary"
+                      onClick={() => {
+                        clearFilter(filter)
+                      }}
+                    >
+                      {filter.clause.name}
```

Comment:
Here we should have `${filter.field.name} ${filter.clause.name} ${filter.value}`